documentation/latest/gsg/ocp.md

```diff
@@ -50,9 +50,13 @@ git clone {{ page.parodos.git_repo }} -b {{ page.parodos.git_branch }}
 
 ### Installing Parodos
 
+Before appling the manifests please make sure you have a [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/)
+configured and set as default. It is needed to bind persistent volume
+claim used by postgresql.
+
```

Comment:
Do you think we can provide a some suggestion/yaml snippet about the easiest way to configure it to let parodos work?

Reply:
Good idea. What do you think about providing a link to https://docs.openshift.com/container-platform/4.13/post_installation_configuration/storage-configuration.html ?